packages/block-editor/src/components/rich-text/index.js

```diff
@@ -293,6 +296,16 @@ class RichTextWrapper extends Component {
 		return formattingControls.map( ( name ) => `core/${ name }` );
 	}
 
+	/**
+	 * Marks the last change as an automatic change at the next idle period to
+	 * ensure all selection changes have been recorded.
+	 */
+	markAutomaticChange() {
+		window.requestIdleCallback( () => {
```

Comment:
Do we polyfill `requestIdleCallback`? It doesn't seem [ready for primetime](https://caniuse.com/#feat=requestidlecallback). Elsewhere in the codebase we have a fallback for it:

https://github.com/WordPress/gutenberg/blob/9c58193621b4bb913bcc6991426c1e9b97dcc9f7/packages/priority-queue/src/index.js#L1

Reply:
Addressed in #17213